storage-clearing/tests.json

```diff
@@ -22,30 +22,51 @@
                 "exceptPlatforms": []
             },
             {
-                "name": "cookie is set on a non-fireproofed subdomain - cookie should be cleared",
-                "cookieDomain": "sub.fireproofed.test",
+                "name": "only subdomain is fireproofed, but cookie is using dot notation - it should not be cleared",
```

Comment:
Konrad, cookie with domain `.no-fire.test` is also sent to `no-fire.test` or `anothersub.no-fire.test`. What it should not me cleared?